tests/spec/core/structure-spec.js

```diff
@@ -52,17 +52,29 @@ describe("Core - Structure", () => {
     };
     ops.config.tocIntroductory = true;
     const doc = await makeRSDoc(ops);
-    const $toc = $("#toc", doc);
-    expect($toc.find("h2").text()).toEqual("Table of Contents");
-    expect($toc.find("> ol > li").length).toEqual(6);
-    expect($toc.find("li").length).toEqual(18);
+    const toc = doc.querySelector("#toc");
+    expect(toc.querySelector("h2").textContent).toEqual("Table of Contents");
+    const heirLI = Array.prototype.slice
```

Comment:
We already have a helper function for this: `utils.children("ol > li");`

Reply:
Need slight help, where should I exactly import utils? :sweat_smile:

Reply:
Thanks!